<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depgraphV::MainWindow</class>
 <widget class="QMainWindow" name="depgraphV::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>windowTile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>625</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Folder</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="rootFolderGroupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>95</height>
           </size>
          </property>
          <property name="title">
           <string>Root folder</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="selectedRootFolder">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pickFolderButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="recursiveCheckBox">
             <property name="text">
              <string>Scan recursively</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="parseGroupbox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>90</height>
           </size>
          </property>
          <property name="title">
           <string>Parse options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="parseHeadersCheckbox">
             <property name="text">
              <string>Parse headers</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parseSourcesCheckbox">
             <property name="text">
              <string>Parse sources</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="headersFilterGroupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Headers filter</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="standardHeaderFilterRadio">
             <property name="text">
              <string>Standard filters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="headersFilterComboBox">
             <item>
              <property name="text">
               <string>*.h</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.hh</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.hxx</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.hpp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.hp</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="customHeadersFilterRadio">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Custom (semicolon-separated)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="headersFilter">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>*.h; *.hh; *.hxx; *.hpp; *.hp</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="sourcesFilterGroupBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Sources filter</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="standardSourcesFilterRadio">
             <property name="text">
              <string>Standard filters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sourcesFilterComboBox">
             <item>
              <property name="text">
               <string>*.cpp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.cc</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.cp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.cxx</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.c++</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*.C</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="customSourcesFilterRadio">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Custom (semicolon-separated)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sourcesFilter">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>*.cpp; *.cc; *.cp; *.cxx; *.c++; *.C</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="drawButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Draw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="depgraphV::Graph" name="graph">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>625</height>
       </size>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::FullViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_as_dot"/>
    <addaction name="actionSave_as_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuRenderer">
     <property name="title">
      <string>&amp;Renderer</string>
     </property>
     <addaction name="actionNative"/>
     <addaction name="actionOpenGL"/>
     <addaction name="actionImage"/>
    </widget>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionSystem_language"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuRenderer"/>
    <addaction name="actionHigh_Quality_Antialiasing"/>
    <addaction name="separator"/>
    <addaction name="menu_Language"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionNative">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Native</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;OpenGL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, Ctrl+O</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, Ctrl+I</string>
   </property>
  </action>
  <action name="actionHigh_Quality_Antialiasing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High Quality &amp;Antialiasing</string>
   </property>
  </action>
  <action name="actionSave_as_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as &amp;image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_as_dot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save as dot...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSystem_language">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>System language</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>depgraphV::Graph</class>
   <extends>QGraphicsView</extends>
   <header>graph.h</header>
   <slots>
    <slot>setHighQualityAntialiasing(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>selectedRootFolder</tabstop>
  <tabstop>pickFolderButton</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
  <tabstop>parseHeadersCheckbox</tabstop>
  <tabstop>parseSourcesCheckbox</tabstop>
  <tabstop>standardHeaderFilterRadio</tabstop>
  <tabstop>headersFilterComboBox</tabstop>
  <tabstop>customHeadersFilterRadio</tabstop>
  <tabstop>headersFilter</tabstop>
  <tabstop>standardSourcesFilterRadio</tabstop>
  <tabstop>sourcesFilterComboBox</tabstop>
  <tabstop>customSourcesFilterRadio</tabstop>
  <tabstop>sourcesFilter</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>drawButton</tabstop>
  <tabstop>graph</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>standardHeaderFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>headersFilterComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standardHeaderFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>headersFilter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standardSourcesFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>sourcesFilterComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standardSourcesFilterRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>sourcesFilter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseSourcesCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sourcesFilterGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseHeadersCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>headersFilterGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHigh_Quality_Antialiasing</sender>
   <signal>triggered(bool)</signal>
   <receiver>graph</receiver>
   <slot>setHighQualityAntialiasing(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseHeadersCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>parseOptionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseSourcesCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>parseOptionsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as_Image</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>saveAsImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as_dot</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>saveAsDot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>exitApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pickFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>depgraphV::MainWindow</receiver>
   <slot>selectRootFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>parseOptionsChanged()</slot>
  <slot>saveAsImage()</slot>
  <slot>saveAsDot()</slot>
  <slot>exitApp()</slot>
  <slot>about()</slot>
  <slot>selectRootFolder()</slot>
 </slots>
</ui>
